app/Model/Cert.php

```diff
@@ -418,6 +428,65 @@ public function beforeSave($options = array())
     }
     $this->data['Cert']['subject'] = trim($this->data['Cert']['subject']);
 
+    if(empty($this->data['Cert']['ordr'])
+       || $this->data['Cert']['ordr'] == '') {
+      // Find the current high value and add one
+      $n = 1;
+
+      // I know the CO Person username
+      $u = !empty($_SESSION["Auth"]["User"]["username"]) ? $_SESSION["Auth"]["User"]["username"] : null;
+      if(!empty($u)) {
+        $oargs = array();
+        $oargs['joins'][0]['table'] = 'identifiers';
+        $oargs['joins'][0]['alias'] = 'Identifier';
+        $oargs['joins'][0]['type'] = 'INNER';
+        $oargs['joins'][0]['conditions'][0] = 'OrgIdentity.id=Identifier.org_identity_id';
+        $oargs['conditions']['Identifier.identifier'] = $u;
+        $oargs['conditions']['Identifier.login'] = true;
+        // Join on identifiers that aren't deleted (including if they have no status)
```

Comment:
Do we need to add a condition for Identifier NOT deleted?

Reply:
This is part of login's query. If it is there i suppose there is a reason. As a result i decided not to experiment.